packages/tldraw/src/lib/ui/components/SharePanel/UserPresenceEditor.tsx

```diff
@@ -44,8 +54,8 @@ export function UserPresenceEditor() {
 
 	return (
 		<div className="tlui-people-menu__user">
-			<UserPresenceColorPicker />
-			{isEditingName ? (
+			{allowChangeColor && <UserPresenceColorPicker />}
```

Comment:
I think it makes sense to have this option but setting it to true results in this

<img width="248" height="140" alt="Image" src="https://github.com/user-attachments/assets/a0929bfd-2614-460e-a798-0529f3a1c040" />

I'd suggest just making the button non-interactive rather than removing it altogether